vault/external_tests/raft/raft_autopilot_test.go

```diff
@@ -126,6 +146,25 @@ func TestRaft_Autopilot_Stabilization_And_State(t *testing.T) {
 	state, err = client.Sys().RaftAutopilotState()
 	require.NoError(t, err)
 	require.Equal(t, []string{"core-0", "core-1", "core-2"}, state.Voters)
+
+	leader := state.Leader
+	deadline := time.Now().Add(2 * time.Minute)
+	for time.Now().Before(deadline) {
```

Comment:
I'm not sure I understand how this test circles back to the addition and use of the reloadable config. Could you clarify a little on this please?

Reply:
This test was failing originally (see first commit).  So it's basically just validating that restarting (or sealing/unsealing) a node doesn't provoke new elections, by virtue of the change to the initial timeout.  We're using a perf multiplier of 5, and the default timeout is 1s, so by sleeping for 10s between seal/unseal we're hoping to trigger (or not, now that we have the fix) an election if the bug is still present.